crates/rpc/src/handlers/block.rs

```diff
@@ -134,3 +146,22 @@ pub async fn get_block_from_id(block_id: ID, db: ReamDB) -> Result<impl Reply, R
         StatusCode::OK,
     ))
 }
+
+/// Called by `debug/beacon/blocks` to get the Beacon Block.
+pub async fn get_beacon_heads(db: ReamDB) -> Result<impl Reply, Rejection> {
+    let beacon_blocks = get_beacon_blocks(&db).await?;
+
+    let mut beacon_heads = Vec::new();
+    for block in beacon_blocks.iter() {
+        let beacon_head = BeaconHeadResponse {
+            root: block.message.tree_hash_root(),
+            slot: block.message.slot,
+            execution_optimistic: false,
+        };
+        beacon_heads.push(beacon_head);
+    }
+    Ok(with_status(
+        BeaconHeadsResponse::json(beacon_heads),
+        StatusCode::OK,
+    ))
+}
```

Comment:
This code is currently printing all headers in the database

Please look at   https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/fork-choice.md#get_head
^ this should give you a rough idea of direction